presto-main/src/main/java/com/facebook/presto/type/TypeRegistry.java

```diff
@@ -161,6 +164,12 @@ public TypeRegistry(Set<Type> types, FeaturesConfig featuresConfig)
         addParametricType(QDIGEST);
         addParametricType(TDIGEST);
 
+        addParametricType(new LongEnumParametricType("test.Hello", new LongEnumType.LongEnumMap(
```

Comment:
This is a production class. I feels like we should add testing related classes only in tests